src/runtime-rs/crates/resource/src/volume/share_fs_volume.rs

```diff
@@ -763,6 +770,20 @@ pub fn generate_mount_path(id: &str, file_name: &str) -> String {
     format!("{}-{}-{}", nid, uid, file_name)
 }
 
+/// This function is used to check whether a given volume is in the allowed copy whitelist.
+/// More specifically, it determines whether the volume's path is located under a predefined
+/// list of allowed copy directories.
+pub(crate) fn is_whitelisted_copy_volume(source_path: &PathBuf) -> bool {
```

Comment:
To try and comply with the Inclusive Naming Initiative which the CNCF created, could we rename whitelist to allowlist here? https://inclusivenaming.org/word-lists/tier-1/whitelist/

Reply:
Fixed, PTAL